### DISCUSSION TITLE
attributes with virtual, in listing show internal Id

### DISCUSSION THREAD

**Discussion by zssvaidar** [atrocore/atrocore#789] [Q&A] [answered]
after enabling virtual option for attribute and in Layouts adding virtual attribute to Product list, it shows it's ID field not label or code

**Comment by rratsun** [comment]
Try to update to the latest version. As I remember, we have already fixed it.

**Reply by zssvaidar** [reply]
in pim or core module?
I am trying to figure out if extensibleAttrobiteOptions can be displayed also as attribute label

**Reply by rratsun** [reply]
Core should be 1.6.8 and more

**Reply by zssvaidar** [reply]
![Screenshot from 2023-07-04 09-09-45](https://github.com/atrocore/atrocore/assets/30953478/2126252d-c169-432c-a4a9-b17f03b125e3)
![Screenshot from 2023-07-04 09-10-08](https://github.com/atrocore/atrocore/assets/30953478/f11c18ec-e5b8-4741-92c3-9be54d416146)
![Screenshot from 2023-07-04 09-10-15](https://github.com/atrocore/atrocore/assets/30953478/6e543051-864c-4714-bdb6-a7227531beab)
Sorry to bother, really but the problem persists on [demo](https://demo.atropim.com/)

**Reply by rratsun** [reply]
ok, we will recheck.

**Reply by rratsun** [reply, marked as answer]
Fixed at PIM 1.9.14
